go_modules/lib/dependabot/go_modules/file_parser.rb

```diff
@@ -78,17 +69,18 @@ def module_info
               end
 
               File.write("go.mod", go_mod_content)
+              File.write("main.go", "package dummypkg\n func main() {}\n")
```

Comment:
Why do we add this here? I think it would be nice to add some comment maybe explaining the reasoning.

Reply:
Yeah good call, we add it so that we can run `go get`, we perform this step in a temp directory where we copy in just the `go.mod` and `go.sum` files, and without the `main.go` we get `go get .: path  is not a package in module rooted at`.

I agree that we should clarify this if we indeed decide to keep it, but if we get rid of `go get`, we can also nuke this step